src/digitalIdentity-frontend/src/app/components/credDefDetail-table-pop-up/credDefDetail-table-pop-up.component.html

```diff
@@ -0,0 +1,14 @@
+<h1>Details to credential definition {{credDef.alias}}</h1>
```

Comment:
use mat-dialog-title, mat-dialog-content, mat-dialog-actions within the dialog template to organize the dialog.

Reply:
Done